test/net/sourceforge/kolmafia/session/QuestManagerTest.java

```diff
@@ -4,19 +4,7 @@
 import static internal.helpers.Networking.assertGetRequest;
 import static internal.helpers.Networking.assertPostRequest;
 import static internal.helpers.Networking.html;
-import static internal.helpers.Player.withAscensions;
-import static internal.helpers.Player.withEffect;
-import static internal.helpers.Player.withEquipped;
-import static internal.helpers.Player.withFamiliar;
-import static internal.helpers.Player.withGender;
-import static internal.helpers.Player.withHandlingChoice;
-import static internal.helpers.Player.withHttpClientBuilder;
-import static internal.helpers.Player.withItem;
-import static internal.helpers.Player.withLastLocation;
-import static internal.helpers.Player.withNextResponse;
-import static internal.helpers.Player.withPasswordHash;
-import static internal.helpers.Player.withProperty;
-import static internal.helpers.Player.withQuestProgress;
+import static internal.helpers.Player.*;
```

Comment:
Don't do this please - your IntelliJ is probably set to do this automatically, but luckily it can also fix it automatically!